libsignal-service/src/envelope.rs

```diff
@@ -129,9 +128,7 @@ impl Envelope {
             source_device: Some(entity.source_device),
             timestamp: Some(entity.timestamp),
             server_timestamp: Some(entity.server_timestamp),
-            source_e164: source.e164(),
```

Comment:
This scares me. A lot.

Reply:
Well, it's really not part of the proto anymore, so it can technically be sent, but can't be used in the official clients anymore.

Reply:
Oh it's totally fine. Just scary.